docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -85,21 +85,23 @@ Please read the [upgrade notes]({{< relref "../installation/upgrading.md#ensure-
 
 General features are included in the Grafana Enterprise edition software.
 
-### Report time range
+### Report grid layout
 
-Report time range can be now customized separately per report, in the report form. By default, the time range from the report's dashboard would apply.
+A new layout option is available when rendering reports: the grid layout. With this option, your report will look more similar to your dashboard by re-using the same panel layout as the one used in the Dashboard.  Learn more about the [grid layout]({{< relref "../enterprise/reporting.md#layout-and-orientation.md" >}}) in the documentation.
```

Comment:
```suggestion
A new layout option is available when rendering reports: the grid layout. With this option your report will use the panel layout from your dashboard so that what you see is what you get.  Learn more about the [grid layout]({{< relref "../enterprise/reporting.md#layout-and-orientation.md" >}}) in the documentation.
```